### ISSUE TITLE
info is correct (updated?)?

### ISSUE THREAD

**Issue by leonpano2006** [rbonghi/jetson_stats#198]
**Describe the bug**
info are not correct
**To Reproduce**
1. open minecraft on jetson and set -Xmx7400m and run
2. open terminal and open jtop
3. then ram is almost full but swap and ram useage is not updated (stucked) but system monitor is keep to update info

**Screenshots**
![image](https://user-images.githubusercontent.com/58368166/157514373-ed201a08-b656-4d6f-aab7-899e607d2fd8.png)
This pictrue is want to show swap usage is freezes on jtop but not system monitor 
![image](https://user-images.githubusercontent.com/58368166/157516746-eddd0b11-3454-4cef-8bf9-5f9917dfc9e1.png)

**Expected behavior**
update info 
**Additional context**

**Board**
 - SOC: [e.g. tegraXXX]
 - ID: [e.g. 10]
 - Board(s): [e.g. P1000-1000]
 - Code Name: [e.g. test]

**Jetpack**
 - Jetpack: [e.g. 4.3]
 - L4T: [e.g. 5.2.1]

**Jetson-Stats**
 - Version: [e.g. 1.8]

**Comment by PrestonHack** [comment]
run jetson_config also set the usergroup for jtop to jetson_stats

**Comment by PrestonHack** [comment]
5 is out? cool

**Comment by rbonghi** [comment]
Hi @PrestonHack ,

I will start to work on jetson-stats soon! Just a few weeks, apologize for my delay

**Comment by rbonghi** [comment]
@leonpano2006 
Does this bug appear on the JP 5.0? if this is true, can you check with the release candidate 4.0.0rc2
```
sudo -H pip install jetson-stats==4.0.0rc2
```

**Comment by leonpano2006** [comment]
I need to fix my synology first

**Comment by rbonghi** [comment]
Now Jetpack added with the latest released **4.0**
```
sudo -H pip install -U jetson-stats
```
